contributors/guide/github-workflow.md

```diff
@@ -17,23 +17,14 @@ description: |
 
 ## 2. Clone fork to local storage
 
-Per Go's [workspace instructions][go-workspace], place Kubernetes' code on your
-`GOPATH` using the following cloning procedure.
+In your shell, define a local working directory as `working_dir`. As [GO Modules] got introduced and became default in GO 1.13, you don't have to put your Go project under `GOPATH`.
```

Comment:
I don't even think you need to mention Go modules here. It's a historical artifact that may cause confusion for someone still new to Go development. I think it would be better and more concise to just keep the first sentence and drop the rest.

Reply:
Thanks for the advise. I think drop it make it more concise. Updated.